docs/modules/prompts/prompt_templates/getting_started.md

```diff
@@ -229,7 +229,11 @@ example_selector = LengthBasedExampleSelector(
     example_prompt=example_prompt, 
     # This is the maximum length that the formatted examples should be.
     # Length is measured by the get_text_length function below.
```

Comment:
here was the problem: "... is measured by the **get_text_length** function below."